RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectrons_cfi.py

```diff
@@ -1,10 +1,48 @@
-from RecoEgamma.EgammaElectronProducers.gsfElectrons_cfi import ecalDrivenGsfElectrons
+import FWCore.ParameterSet.Config as cms
 
-lowPtGsfElectrons = ecalDrivenGsfElectrons.clone(gsfElectronCoresTag = "lowPtGsfElectronCores")
+from RecoEgamma.EgammaTools.regressionModifier_cfi import regressionModifier106XUL
 
-from Configuration.Eras.Modifier_fastSim_cff import fastSim
-fastSim.toModify(lowPtGsfElectrons,ctfTracksTag = "generalTracksBeforeMixing")
+_lowPtRegressionModifier = regressionModifier106XUL.clone(
+    modifierName = 'EGRegressionModifierV3',
+    rhoTag = 'fixedGridRhoFastjetAll',
+    eleRegs = dict(
+        ecalOnlyMean = dict(
+            lowEtHighEtBoundary = 20.,
+            ebLowEtForestName = ":lowPtElectron_eb_ecalOnly_05To50_mean",
+            ebHighEtForestName = ":lowPtElectron_eb_ecalOnly_05To50_mean",
+            eeLowEtForestName = ":lowPtElectron_ee_ecalOnly_05To50_mean",
+            eeHighEtForestName = ":lowPtElectron_ee_ecalOnly_05To50_mean",
+            ),
+        ecalOnlySigma = dict(
+            lowEtHighEtBoundary = 20.,
+            ebLowEtForestName = ":lowPtElectron_eb_ecalOnly_05To50_sigma",
+            ebHighEtForestName = ":lowPtElectron_eb_ecalOnly_05To50_sigma",
+            eeLowEtForestName = ":lowPtElectron_ee_ecalOnly_05To50_sigma",
+            eeHighEtForestName = ":lowPtElectron_ee_ecalOnly_05To50_sigma",
+            ),
+        epComb = dict(
+            ecalTrkRegressionConfig = dict(
+                lowEtHighEtBoundary = 20.,
+                ebLowEtForestName = ":lowPtElectron_eb_ecalTrk_05To50_mean",
+                ebHighEtForestName = ":lowPtElectron_eb_ecalTrk_05To50_mean",
+                eeLowEtForestName = ":lowPtElectron_ee_ecalTrk_05To50_mean",
+                eeHighEtForestName = ":lowPtElectron_ee_ecalTrk_05To50_mean",
+                ),
+            ecalTrkRegressionUncertConfig = dict(
+                lowEtHighEtBoundary = 20.,
+                ebLowEtForestName = ":lowPtElectron_eb_ecalTrk_05To50_sigma",
+                ebHighEtForestName = ":lowPtElectron_eb_ecalTrk_05To50_sigma",
+                eeLowEtForestName = ":lowPtElectron_ee_ecalTrk_05To50_sigma",
+                eeHighEtForestName = ":lowPtElectron_ee_ecalTrk_05To50_sigma",
+                ),
+        )
+    ),
+)
 
-from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
-pp_on_AA.toModify(lowPtGsfElectrons.preselection, minSCEtBarrel = 15.0)
-pp_on_AA.toModify(lowPtGsfElectrons.preselection, minSCEtEndcaps = 15.0)
+lowPtGsfElectrons = cms.EDProducer("LowPtGsfElectronFinalizer",
+                                   previousGsfElectronsTag = cms.InputTag("lowPtGsfElectronsPreRegression"),
+                                   regressionConfig = _lowPtRegressionModifier,
+)
```

Comment:
following https://github.com/cms-sw/cmssw/pull/32391#discussion_r541460175

```suggestion
from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronFinalizer_cfi import lowPtGsfElectronFinalizer
lowPtGsfElectrons = lowPtGsfElectronFinalizer.clone(
                                   previousGsfElectronsTag = "lowPtGsfElectronsPreRegression",
                                   regressionConfig = _lowPtRegressionModifier,
)
```

Reply:
Done.